orca-queue/src/main/kotlin/com/netflix/spinnaker/orca/q/handler/RunTaskHandler.kt

```diff
@@ -305,7 +305,6 @@ class RunTaskHandler(
 
   private fun Stage.withLoggingContext(taskModel: com.netflix.spinnaker.orca.pipeline.model.Task, block: () -> Unit) {
     try {
-      MDC.put("application", this.execution.application)
```

Comment:
this is now included automatically here https://github.com/spinnaker/orca/pull/2861/files#diff-574feb3cd5fd21d4373f7a819aa5a401R65